UBC-Meetable/code
# Android choosing major debug

## Objective
Debugging the android sign up page so that the keyboard shows up correctly

Implements/Fixes CH####
adding the line below and importing platform from react 
behavior={Platform.OS === "ios" ? "padding" : "height"}

In React Native website it is mentioned: 
"Specify how to react to the presence of the keyboard. On both iOS and Android, setting behavior is recommended."
Adding this line solves the problem for an Android user, at least for me, but it should be checked so that it does not effect the iOS users.

## Screenshots

Before:
Keyboard shows up one second and then closes

After:
![image](https://user-images.githubusercontent.com/81648012/140616770-c69ecd56-7d5f-45a4-b0b7-43e557f5e2a4.png)

## Testing instructions
Change the behaviour and see how the keyboard behaviour changes over both platforms


## Important
The change must be checked on iOS before merging as unwanted changes are possible


## Checklist
- [ ] Ensure all tests pass; iOS should be tested
- [ ] Verify changes work as expected in the deploy preview environment, it works for android
